contracts/Swap/SwapRouter.sol

```diff
@@ -717,4 +695,31 @@ contract SwapRouter is Ownable2Step, RouterHelper, IPancakeSwapV2Router {
             revert RepayError(msg.sender, vTokenAddress, response);
         }
     }
+
+    /**
+     * @notice check for min amount out.
+     * @param path The addresses of the underlying token
+     * @param balanceBefore Balance before the swap.
+     * @param amountOutMin Min amount out threshold.
```

Comment:
I miss the `to` param doc. The `@notice` fragment could be: check if the balance of `to` minus the balanceBefore is greater than the amountOutMin

Reply:
Fixed, 6ef9faf8fc19a066c7e4a7dac440a9bd25f2a1cc